shared/src/helpers/date/date.ts

```diff
@@ -1 +1,3 @@
 export { formatISO as getFormattedISODate } from 'date-fns';
+export { format as getFormattedDate } from 'date-fns';
```

Comment:
move it to `shared/src/helpers/date/get-formatted-date/get-formatted-date.helper.ts`

```
getFormattedDate(date: Date, formatType: DateFormatType)
```

`DateFormatType` should be enum